apps/etterlatte-saksbehandling-ui/client/src/shared/api/oppgaverny.ts

```diff
@@ -33,37 +34,70 @@ export type Oppgavetype =
   | 'UNDERKJENT'
   | 'GOSYS'
 
-export const erOppgaveRedigerbar = (status: Oppgavestatus, type: string): boolean =>
-  ['NY', 'UNDER_BEHANDLING'].includes(status) && !['GOSYS'].includes(type)
+export const erOppgaveRedigerbar = (status: Oppgavestatus): boolean => ['NY', 'UNDER_BEHANDLING'].includes(status)
 
 export const hentNyeOppgaver = async (): Promise<ApiResponse<OppgaveDTOny[]>> => apiClient.get('/nyeoppgaver')
 
 export interface SaksbehandlerEndringDto {
   saksbehandler: string
+  versjon: string | null
```

Comment:
Siden dette er egentlig går mot to forskjellige systemer kan det være greit å skille denne og de andre ut i egne DTOer som kun går mot Gosys.

Reply:
done